tests/run_tests.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+# Test runner to test the following examples:
+# bounds.c, set_bounds.c, general_bounds.c, xor_pointers.c.
+#
+# *** NOTE ***
+# The following examples do not need to be
+# tested because they are simply exploring features or
+# they are tests themselves/experiments:
+# allocate.c, check_length.c, check_mask.c, function.c,
+# seal.c, sentry.c, setjmp.c, stackscan.c
+set -e
+
+: "${BUILD_DIR:="../bin"}"
+: "${SSHPORT:=10021}"
+# Examples that must trigger an "In-address space security exception"
+EXAMPLES="bounds set_bounds general_bounds xor_pointers"
+ARCH=$(ssh -p $SSHPORT -t -o LogLevel=QUIET root@127.0.0.1 uname -m)
+
+for example in ${EXAMPLES}; do
+    if ! [ $(find $BUILD_DIR -prune -empty 2>/dev/null) ]; then
+        if ! [ -z "${ARCH##*riscv*}" ] && [ "$example" = "set_bounds" ]; then
+            echo "Skipping '$example' because you are not on riscv64..."
+            continue
+        fi
+        scp -P $SSHPORT "$BUILD_DIR/$example" root@127.0.0.1:/root
+        exit_status=0
+        # Put the argument, when not needed will be simply ignored
```

Comment:
What does "put the argument" mean?

Reply:
A personal mnemonic that should have been removed.